metagraph/src/cli/config/config.hpp

```diff
@@ -120,6 +119,8 @@ class Config {
     size_t alignment_max_seed_length = std::numeric_limits<size_t>::max();
     size_t alignment_max_num_seeds_per_locus = std::numeric_limits<size_t>::max();
 
+    double alignment_fraction_of_top = 0.8;
```

Comment:
Rename, when looking at this name, I'd think this is about keeping 80% of the best paths at each step, which is not true. So I'd be confused.